docs/src/components/ThemeSelect.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useThemeSwitch } from './ThemeSwitch';
+import { useStyles } from '@marigold/system';
+import unicornTheme from '@marigold/theme-unicorn';
+import b2bTheme from '@marigold/theme-b2b';
+
+export const themes = {
+  b2bTheme,
+  unicornTheme,
+};
+
+const themeSelectStyles = {
+  position: 'absolute',
+  right: b2bTheme.space.none,
+  padding: '8px',
+  marginTop: '4px',
+  marginRight: '12px',
+  background: '#f6f8fa',
+  color: b2bTheme.colors.gray60,
+  border: 'none',
+  cursor: 'pointer',
+  fontSize: b2bTheme.fontSizes.xxsmall,
+  fontFamily: b2bTheme.fonts.body,
+};
+
+export const ThemeSelect: React.FC = () => {
+  const { current, themes, setTheme } = useThemeSwitch();
+
+  return (
+    <select
```

Comment:
Why are you not using Marigold's `<Select>` component?

Reply:
Our Select doesn't like it should. We have to rewrite some things. Is this e separate issue our should I do this here?

Reply:
That's why we are dogfooding! I've done the same thing #1132 ... I wanted to use our `<Link>` component but it didn't work. That's our chance to fix this!

Reply:
New issue/PR to rewrite the Select component